core/domain/suggestion_services.py

```diff
@@ -192,8 +192,9 @@ def mark_review_completed(suggestion, status, reviewer_id):
             are STATUS_ACCEPTED or STATUS_REJECTED.
         reviewer_id: str. The ID of the user who completed the review.
     """
-    if(status not in [suggestion_models.STATUS_ACCEPTED,
-                      suggestion_models.STATUS_REJECTED]):
+    if(
+            status not in [suggestion_models.STATUS_ACCEPTED,
```

Comment:
Probably can move to previous line and drop outer paren?

(As an aside, there should have been a space after "if" and before that paren. Was there a lint check for it?